### DISCUSSION TITLE
Safe to use?

### DISCUSSION THREAD

**Discussion by Evelynox** [project64/project64#2227] [Support]
Everytime I download and install a programm on my PC I test it for viruses on a site called "VirusTotal.com", the site runs the file through many Anti-Virus applications, I do it because a older PC of mine got hacked because I wasn't carefull enough and so I did with the Zip file of Project64 and it found a Trojan, but when I extracted the folder and let the extracted folder run through the site it sayed the folder was safe so I'm not sure why that is, so I'm a little sceptic, because I don't want to have a Virus on one of my computers again. I would appreciate anyone who can explain why and if someone can assure me from his perspective that it's 100% safe to use.

![Bild_2022-04-22_174729420](https://user-images.githubusercontent.com/60473108/164749432-47492978-5fa8-4365-81d1-c756c555cada.png)

**Comment by project64** [comment]
it is always good to run through virus total, Yes Project64 is safe.

It looks like a false positive, it is 1 out of 61 and something generic

**Comment by molecula451** [comment]
If not the SAFER to use, out there, if you're in doubt you can always download the repo and build it up by yourself
